backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -118,6 +118,13 @@
           </div>
         </div>
 
+        <div class="col-xs-12 col-md-4">
+          <div class="form-group">
+            <%= label_tag :q_channel_eq, Spree.t(:channel) %>
+            <%= f.select :channel_eq, Spree::Order.pluck('DISTINCT channel'), { include_blank: true }, class: 'select2 js-filterable' %>
```

Comment:
Can we move `Spree::Order.pluck()` to a helper method? 
This kind of logic shouldn't be present in the views 🙂

Reply:
I do agree with you that this kind of logic shouldn't be present in the views, but look at the whole file: https://github.com/spark-solutions/spree/blob/6ca551ae307e904c57f61a587406a7452ce361cd/backend/app/views/spree/admin/orders/index.html.erb
In my opinion we either should refactor the whole file, or leave as is to be consistent - when it's always easier to read.

Reply:
Makes sense, we should make an issue to refactor views some day in the future 👍